tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -172,11 +236,56 @@ def test_45519(self):
         LOGGER.info("STARTED: Test Delete source object when copy simple object is in progress in "
                     "the same bucket")
         self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
-                                                         "dest-obj", "NoSuchKey"),
+                                                         "dest-obj", errmsg.NO_SUCH_KEY_ERR),
                                                         (self.srcbuck, "src-obj"))
         LOGGER.info("ENDED: Test Delete source object when copy simple object is in progress in "
                     "the same bucket")
 
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45520")
+    @CTFailOn(error_handler)
+    def test_45520(self):
+        """Test Delete destination object when copy simple object is in progress in the same
+        bucket"""
+        LOGGER.info("STARTED: Test Delete destination object when copy simple object is in "
+                    "progress in the same bucket")
+        self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
+                                                         "dest-obj", None),
+                                                         (self.srcbuck, "dest-obj"))
+        LOGGER.info("ENDED: Test Delete destination object when copy simple object is in "
+                    "progress in the same bucket")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45521")
+    @CTFailOn(error_handler)
+    def test_45521(self):
+        """Test Delete destination simple copied object with the same name as source object
+        in the same bucket"""
+        LOGGER.info("STARTED: Test Delete destination simple copied object with the same name "
+                    "as source object in the same bucket")
+        self.create_put_parallel_copy_and_delete_object((self.srcbuck, "obj1", self.srcbuck,
+                                                         "obj1", errmsg.INVALID_REQ_ERR,
+                                                         errmsg.NOT_FOUND_ERRCODE), (self.srcbuck,
```

Comment:
We can move the second tuple being passed to a new line here. Same holds for the flag being passed in line 271. Let's move it to a new line as well.

Applicable to other calls to create_put_parallel_copy_and_delete_object/bucket as well.

Reply:
done